ons_alpha/release_calendar/models.py

```diff
@@ -126,6 +138,11 @@ def get_context(self, request, *args, **kwargs):
         context["related_links"] = self.related_links_for_context
         context["toc"] = self.toc
 
+        # Explicitly convert self.content to a list to ensure it's iterable
```

Comment:
Question: When is the content not iterable? You should just need the `include_block` in the template.